addons/pos_restaurant/static/src/app/floor_screen/floor_screen.js

```diff
@@ -384,6 +384,11 @@ export class FloorScreen extends Component {
         if (!confirmed) {
             return;
         }
+        if (!newName) {
```

Comment:
What do you think about disabling the `Confirm` button on the popup if there is no name? The popup could have a new prop ( something like `isValid` ) that would be a function that takes the `inputValue` and returns a boolean that could be used to disable the button. 
This way you never have to throw the `ErrorPopup`

Reply:
thanks for the review and feedback @vlst-odoo 
will update the PR as suggested.

in case if you have time, can you review this too

https://github.com/odoo/odoo/pull/132132
https://github.com/odoo/odoo/pull/131992